src/applications/mhv-medical-records/util/helpers.js

```diff
@@ -303,10 +303,7 @@ export const parseDate = str => {
     return str;
```

Comment:
I know that this PR is modifying the existing function rather than adding it, but I wonder if this would be a good opportunity to change the `parseDate` name? It's ambiguous with [other utils](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/mhv/downtime/utils/date.js#L29) and also doesn't seem well named since it parses and formats. A docblock would also be nice 🙂